Sources/PDFXKitObjC/Private/PDFXView+PSPDFKit.h

```diff
@@ -18,8 +18,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (PSPDFViewController *)makePSPDFViewController;
 
-- (void)pspdfViewControllerDidChangePageNotification:(NSNotification *)notification NS_REQUIRES_SUPER;
-
```

Comment:
Seems to be a convention here to put the private method in a header. I didn’t replicate that with this change, but I could.

Reply:
I don't think that's needed. This seems fine as is.